### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Gerghet**
Sonarlint errors fixed

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
Comment (marcell-balogh) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:33]: nice throw

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -40,7 +40,7 @@ public boolean fire(int numberOfTorpedos){
 
     if (r >= FAILURE_RATE) {
       // successful firing
-      this.torpedoCount =- numberOfTorpedos;
+      this.torpedoCount -= numberOfTorpedos;
```
Comment (marcell-balogh) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:43]: very good fix

**Review by marcell-balogh — COMMENTED** [review]
very good idea

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 2 above)
Comment (Gerghet) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:43]: thx i worked on this for hours

**Review by marcell-balogh — APPROVED** [review]

**Review by marcell-balogh — APPROVED** [review]

**Pull request merged**